x/group/server/invariants.go

```diff
@@ -0,0 +1,91 @@
+package server
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/regen-network/regen-ledger/orm"
+	"github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func (s serverImpl) RegisterInvariantsHandler(ir sdk.InvariantRegistry) {
+	ir.RegisterRoute(group.ModuleName, "Tally-Votes", s.TallyVotesInvariant())
+}
+
+func (s serverImpl) AllInvariants() sdk.Invariant {
```

Comment:
is this method really needed?

Reply:
Yeah, `AllInvariants` is the function that runs all the invariant's functions included in our module

Reply:
from where it's called?

Reply:
I don't think it's called anywhere, if `tallyVotesInvariant` is the only invariant in our module, then we can safely remove this function

Reply:
I don't think it's called anywhere, if `tallyVotesInvariant` is the only invariant in our module, then we can safely remove this function